### PR TITLE
Closed Eyes

### PR DESCRIPTION
**Pull request by lectronyx**
Adds the "Closed Eyes" marking from Vorestation, because Sergal Eyes suck ass and in this house we bully cheeses

## About The Pull Request

More customization options from Vorestation! This one should be good for characters with animu-esque "closed eyes" or two pairs of eyes aligned horizontally.

Shit might be a little borked, and the reason for which genuinely baffles me.

## Why It's Good For The Game

Customization's good, bro.

## Changelog
:cl:
add: closed eyes pattern to eye_stuff.dm
imageadd: closed eyes state to eye_stuff.dmi
/:cl:

### CODE REVIEW

**Comment by lectronyx** [comment]
baby's first PR

File: code/modules/mob/new_player/sprite_accessories/markings/eye_stuff.dm [hunk 1]
```diff
@@ -52,3 +52,8 @@
 	name = "Third Eye"
 	icon_state = "thirdeye"
 	body_parts = list(BP_HEAD)
+
+/datum/sprite_accessory/marking/eye_stuff/closedeye
+	name = "Closed Eyes"
+	icon_state = "closedeyes"
+	body_parts = list(BP_HEAD)
```
Comment (TheLordME) [inline code/modules/mob/new_player/sprite_accessories/markings/eye_stuff.dm:59]: You need to name the state **in** the .dmi "closedeyes-head"
here in the .dm it stays "closedeyes"

**Review by TheLordME — CHANGES_REQUESTED** [review]

File: code/modules/mob/new_player/sprite_accessories/markings/eye_stuff.dm (hunk 1 above)
Comment (lectronyx) [reply code/modules/mob/new_player/sprite_accessories/markings/eye_stuff.dm:59]: What I'm hearing is that I just need to give a little -head

**Comment by lectronyx** [comment]
silicons please give me some love here i am begging you

**Pull request merged**